v2.0/drop-sequence.md

```diff
@@ -28,12 +28,12 @@ table td:first-child {
 -----------|------------
 `IF EXISTS` |  Drop the sequence only if it exists; if it does not exist, do not return an error.
 `sequence_name` | The name of the sequence you want to drop. Find the sequence name with `SHOW CREATE TABLE` on the table that uses the sequence.
-`CASCADE` / `RESTRICT` | Not yet implemented. Currently, you can drop a sequence even if it has a dependency.
+`RESTRICT` | _(Default)_ Do not drop the sequence if any objects (such as [constraints](constraints.html) and tables) depend on it.
+`CASCADE` | Not yet implemented. Currently, you can only drop a sequence if nothing depends on it.
 
-<!-- `CASCADE` > Drop all objects (such as [constraints](constraints.html) and tables) that depend on the sequence.<br><br>`CASCADE` does not list objects it drops, so should be used cautiously.
-`RESTRICT` > _(Default)_ Do not drop the sequence if any objects (such as [constraints](constraints.html) and tables) depend on it. -->
+<!-- `CASCADE` > Drop all objects (such as [constraints](constraints.html) and tables) that depend on the sequence.<br><br>`CASCADE` does not list objects it drops, so should be used cautiously. -->
 
-<!-- ## Examples
+## Examples
 
 ### Remove a Sequence (No Dependencies)
```

Comment:
I'd change `No Dependencies` to `No Dependents`.

Reply:
The `SELECT * FROM information_schema.sequences` queries need closing semicolons.

Also, you might want to add a "List All Sequences" example to `CREATE SEQUENCES`. Is querying `information_schema.sequences` the only way?

Reply:
Done.

Currently `information_schema.sequences` is the only way to list all sequences.